timeless/restaurants/models.py

```diff
@@ -113,19 +113,11 @@ class Table(PosterSyncMixin, DB.Model):
     shape_id = DB.Column(DB.Integer, DB.ForeignKey("table_shapes.id"))
     min_capacity = DB.Column(DB.Integer, DB.ForeignKey("scheme_types.id"))
     deposit_hour = DB.Column(DB.Integer, DB.ForeignKey("scheme_types.id"))
-    created = DB.Column(DB.DateTime, nullable=False)
-    updated = DB.Column(DB.DateTime, nullable=False)
 
     reservations = DB.relationship("TableReservation", back_populates="table")
 
     DB.UniqueConstraint(u"name", u"floor_id")
 
-    @validate_required("name", "x", "y", "width", "height", "status",
```

Comment:
@vryazanov We do not need this anymore ?

Reply:
@vladarefiev No, we don't. It breaks logic of forms. Form at first creates an instance without any args in constructor (and on this step validator raises an exception) and only then sets attributes.